sdk/schema/validation.go

```diff
@@ -136,3 +136,14 @@ func IsStringSliceASet(slice []string) bool {
 
 	return true
 }
+
+func UnnecessaryPluginNameDefined(plugin Plugin) bool {
```

Comment:
nit: let's use a more suggestive function name
```suggestion
func RedundantPluginNameDefined(plugin Plugin) bool {
```